tests/unittest/_torch/attention/sparse/deepseek_v4/test_deepseek_v4_sparse_mla.py

```
@@ -0,0 +1,1649 @@
+                cache_manager,
+                li,
+                gen_request_ids,
+                head_dim,
+                device,
+                gen_kv_lens,
+                compress_ref_data[li],
+            )
+
+    # 4. Mixed metadata: request 0 = context, requests 1..N = generation.
+    mixed_seq_lens = [context_lengths[0]] + [generation_seq_len_q] * num_gen
+    mixed_cached_lens = [0] + gen_cached_lens
```

Ruff (0.15.18)

[warning] Consider `[0, *gen_cached_lens]` instead of concatenation

Replace with `[0, *gen_cached_lens]`

(RUF005)